gradle/init-scripts/build-scan.init.gradle.kts

```diff
@@ -27,10 +27,5 @@ fun configureExtension(extension: Any) {
         } else {
             "publishAlways"()
         }
-        setProperty("server", "https://e.grdev.net")
-
-        if (!System.getProperty("slow.internet.connection", "false").toBoolean()) {
-            setProperty("captureTaskInputFiles", true)
```

Comment:
Why remove this? I don't see the plugin activating `captureTaskInputFiles`.

Reply:
It's here: https://github.com/gradle/gradle-enterprise-conventions-plugin/blob/9e0fc6545148014273ef2fd3e1c94beacfb7fb54/src/main/java/com/gradle/enterprise/conventions/GradleEnterpriseConventionsPlugin.java#L64

I don't know if `slow.internet.connection` is still useful.

Reply:
I don't know how bad this affects folks when using background upload. Maybe @melix has some input.

Reply:
Slow internet connection is for downloads from the cache, when it's not worth downloading. For upload it's not a problem in my experience thanks to the async upload.